bench/sleigh/CMakeLists.txt

```
@@ -0,0 +1,22 @@
+cmake_minimum_required(VERSION 3.5)
+project(bench-sleigh)
+
+add_subdirectory("../../libs/sleigh" "sleigh")
+
+# The load_bin helpers are written in C and can't be included in the C++ benchmark code
+#
+# To use the helpers, we've created a C library called load_bin and wrapped it with a C++ compatible
+# header (`load_bin.h`)
+set("load-bin-files" "../load_bin.inc")
+add_library("load-bin" ${load-bin-files})
```

I think to prevent having to copy the Windows DLL after building, you can add `STATIC` to this library to always build it statically.
```suggestion
add_library("load-bin" STATIC ${load-bin-files})
```

---

[1abcecb](https://github.com/athre0z/disas-bench/pull/7/commits/1abcecb0ff0a4bf3fb475bbd1a5466f820e20fcb)